graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/NodeHelper.java

```diff
@@ -156,11 +156,12 @@ private static boolean hasPropertyValue(JCRNodeWrapper node, String language, St
      * @throws RepositoryException in case of JCR access errors
      */
     public static JCRNodeWrapper getNodeInLanguage(JCRNodeWrapper node, String language) throws RepositoryException {
-        if (language == null) {
+        Locale sessionLocale = node.getSession().getLocale();
+        if (language == null && sessionLocale == null) {
             return node;
         }
         String workspace = node.getSession().getWorkspace().getName();
-        Locale locale = LanguageCodeConverters.languageCodeToLocale(language);
+        Locale locale = language == null ? sessionLocale : LanguageCodeConverters.languageCodeToLocale(language);
```

Comment:
This looks strange. if language == null, then we use the locale of the session of the current node, so the current node is already in the current locale ? 
I dont understand why is this necessary ?